examples/Reporting/GetHotelAdsPerformance.php

```diff
@@ -130,7 +130,7 @@ public static function runExample(GoogleAdsClient $googleAdsClient, $customerId)
                 $googleAdsRow->getAdGroup()->getId()->getValue(),
                 $googleAdsRow->getCampaign()->getId()->getValue(),
                 $googleAdsRow->getHotelCheckInDayOfWeek(),
```

Comment:
This too. I think your IDE  should warn this, no?

Reply:
Done.